src/routes/blogs.rs

```diff
@@ -95,6 +95,11 @@ pub fn new(rockets: PlumeRocket, _user: User) -> Ructe {
     ))
 }
 
+#[get("/validate/<validation_id>")]
+pub fn validate(validation_id: String, rockets: PlumeRocket) -> Result {
```

Comment:
the name mismatch the one in main.rs, and Result take 2 type arguments (probably both String I think)

Reply:
thanks / sorry for the noise 🙇‍♀️

the two files were edited with about a week or two in between 😅

Reply:
np :wink: